## Enables yorick-mode even if yorick is not yet loaded

diff -urN orig/emacs/yorick-auto.el debian/emacs/yorick-auto.el
--- orig/emacs/yorick-auto.el   2007-08-20 09:13:52.000000000 +0200
+++ debian/emacs/yorick-auto.el 2007-08-20 09:15:43.000000000 +0200
@@ -4,6 +4,7 @@
 (autoload 'yorick "yorick" "see yorick.el in site-lisp dir" t)
 (autoload 'ysh "yorick" "see yorick.el in site-lisp dir" t)
 (autoload 'yssh "yorick" "see yorick.el in site-lisp dir" t)
+(autoload 'yorick-mode "yorick" "see yorick.el in site-lisp dir" t)
 (autoload 'yutil-setup "yorick" "see yorick.el in site-lisp dir" nil)
 (if (assoc "\\.i$" auto-mode-alist)
     ;; this presumes user doesn't want .i files to be in C mode (default)
